libraries/AP_Scripting/examples/aux_cached.lua

```diff
@@ -0,0 +1,37 @@
+--[[ 
+   example for getting cached aux function value
+--]]
+
+local RATE_HZ = 10
+
+local MAV_SEVERITY_ERROR = 3
+local MAV_SEVERITY_INFO = 6
+
+local AUX_FUNCTION_NUM = 302
+
+local last_func_val = nil
+local last_aux_pos = nil
+
+function update()
+   local aux_pos = rc:get_aux_cached(AUX_FUNCTION_NUM)
```

Comment:
I would like a plane for when we can di:
```
   local aux_pos = aux:get_cached(AuxFunc::Number)
```

(or whatever).

Do we put stub functions in?  Forever-more? 

Just want a plan, this isn't a blocker.

Reply:
We could do it as a manual binding to allow us to give it a custom name for now and change it when the aux split happens

Reply:
we could make that possible if/when we have aux:

Reply:
One its in `rc` were stuck with it in `rc` unless we make a braking change, something we have avoided thus far.

Reply:
I don't think it is terrible it being in rc, but if/when we added aux we could make it available in both
note that we have the same problem already with rc:run_aux_function